app/api/helpers/system_mails.py

```diff
@@ -111,7 +111,8 @@
         'recipient': 'User',
         'subject': u'{app_name}: Password Reset',
         'message': (
-            u"Please use the following link to reset your password.<br> <a href='{link}' target='_blank'>{link}</a>"
+            u"Please use the following link to reset your password.<br> <a href='{link}' target='_blank'>{link}</a>" +
+            " Or copy this token to your Eventyay App: {token} "
```

Comment:
I'm not confident if we should say "Eventyay" as specific on email template. May be we can use `{app_name}` here?